test/integration/search/search_test.rb

```diff
@@ -473,6 +473,31 @@ def test_show_the_query
     assert parsed_response.fetch("elasticsearch_query")
   end
 
+  def test_dfid_can_search_by_every_facet
```

Comment:
Yes, it did occur that this was duplication – you're right, this is more or less just scaffolding to avoid a contemporaneous problem. You want another PR to move it? I'm not sure what "nvm" stands for ;)